package.json

```diff
@@ -40,10 +41,10 @@
     "react-emoji-render": "^2.0.1",
     "react-flagpack": "^2.0.5",
     "react-hook-form": "^7.53.0",
+    "react-hotkeys-hook": "^4.6.1",
```

Comment:
Can use our current lib https://streamich.github.io/react-use/?path=/story/sensors-usekey--docs

Reply:
Not working for hotkeys it's only for for 1 key